pkg/plugin/v2/api.go

```diff
@@ -131,6 +131,14 @@ func (p *createAPIPlugin) Validate() error {
 		return err
 	}
 
+	if p.resource.Group == "" && p.config.Domain == "" {
+		return fmt.Errorf("can not have group and domain both empty")
+	}
+
+	if p.config.IsV2() && p.config.Domain == "" {
+		fmt.Errorf("group cannot be empty")
```

Comment:
we cannot do it for v2.

Reply:
We can not validate the group length based on the version at options level. The type Options do not have the information about the project config and we need project config to determine the project is V2 or V3

Reply:
Fixed it

Reply:
You can close this one it is solved 👍